docs/source/conf.py

```diff
@@ -13,13 +13,14 @@
 import os
 import sys
 sys.path.insert(0, os.path.abspath('.'))
-
+from datetime import datetime
 
 # -- Project information -----------------------------------------------------
 
 project = 'Global-workflow'
-copyright = '2023, Kate Friedman, Walter Kolczynski, Rahul Mahajan, Lin Gan, Arun Chawla'
-author = 'Kate Friedman, Walter Kolczynski, Rahul Mahajan, Lin Gan, Arun Chawla'
+year = datetime.now().year
+copyright = f"2015-{year} NOAA/NWS/NCEP/EMC, Kate Friedman, Walter Kolczynski, Rahul Mahajan, Lin Gan"
```

Comment:
```suggestion
copyright = f"2015-{year} NOAA/NWS/NCEP/EMC"
```
I don't think the individuals can hold any copyright (besides, they wouldn't want to for legal reasons)